Lib/vanilla/test/testTools.py

```diff
@@ -39,8 +39,13 @@ def executeVanillaTest(cls, nibPath=None, calls=None, **kwargs):
 
         editMenuItem = NSMenuItem.alloc().initWithTitle_action_keyEquivalent_("Edit", None, "")
         editMenu = NSMenu.alloc().initWithTitle_("Edit")
+        editMenu.addItemWithTitle_action_keyEquivalent_('Cut', NSText.cut_, 'x')
```

Comment:
Please you double quotes (`"`) and write the selectors as strings, such as `"cut:"`, as per main discussion.